content/src/main/content/jcr_root/apps/acs-commons/components/authoring/templatedtextfield/render.jsp

```diff
@@ -0,0 +1,160 @@
+<%--
+  ~ #%L
+  ~ ACS AEM Commons Bundle
+  ~ %%
+  ~ Copyright (C) 2019 Adobe
+  ~ %%
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~ #L%
+  --%><%
+%><%@include file="/libs/granite/ui/global.jsp" %><%
```

Comment:
I know that you are taking the approach provided by the product itself, but personally I don't like these JSPs (I wonder why TouchUI uses them at all ....).
But anyway, I would appreciate it if you could rewrite this file to use HTL and a SlingModel.